docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Use the devfile specification to develop a node.js “Hello World” application. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to create a Kubernetes cluster on your local machine. To create such a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube].
+* To access your namespaces, download link:https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/[kubectl].
+* When you use minikube for the first time, you must enable Ingress to access your devfile projects:
+** Set up link:https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/[Ingress on minikube].
```

Comment:
The two sub-points seem to do the same thing. One points to docs on how to set up Ingress, the other tells you how to do it. So, which one do we want? Or combine them?